x/ccv/provider/keeper/power_shaping.go

```diff
@@ -337,13 +333,16 @@ func (k Keeper) SetConsumerPowerShapingParameters(ctx sdk.Context, consumerId st
 
 	store.Set(types.ConsumerIdToPowerShapingParametersKey(consumerId), bz)
 
-	// update allowlist and denylist indexes if needed
+	// update allowlist, denylist and priority indexes if needed
```

Comment:
nit: s/priority/prioritylist to be consistent with allowlist and denylist

Reply:
It's fixed now.